spec/requests/api/conditions_spec.rb

```diff
@@ -124,7 +124,7 @@ def assign_conditions_to(resource)
 
       expect(response.parsed_body["results"].count).to eq(2)
 
-      expect(Condition.pluck(:description)).to eq(%w(change change2))
+      expect(Condition.pluck(:description)).to contain_exactly("change", "change2")
```

Comment:
Elsewhere we've been using `match_array`, so perhaps prefer that.  (Why oh why does RSpec keep adding new methods that do the same thing ☹️ )